backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/InvoiceCandidateHandlerBL.java

```diff
@@ -302,11 +302,12 @@ private ImmutableList<I_C_Invoice_Candidate> createForModel(
 			final IInvoiceCandidateHandler invoiceCandiateHandler)
 	{
 		final PerformanceMonitoringService performanceMonitoringService = SpringContextHolder.instance.getBean(PerformanceMonitoringService.class);
-		final SpanMetadata request = SpanMetadata.builder()
-				.type("createMissingInvoiceCandidates")
+		final Metadata request = Metadata.builder()
```

Comment:
can we refactor this as a constant?

Reply:
for this metric type currently no metrics get created and it's no visualisation realized. So i removed it